centos_ci.sh

```diff
@@ -56,6 +55,22 @@ function install_core_deps() {
   echo 'CICO: Core dependencies installed'
 }
 
+function install_golang() {
+  # Install virt7-container-common-candidate to install latest golang package
```

Comment:
This is the same repo all the container specific ecosystem use like podman, buildah ..etc.